lib/transport/abstract-transport.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var util = require("util");
+var EventEmitter = require('events');
+
+var AbstractTransport = function () {
+	if (this.constructor === AbstractTransport) {
+		throwBecauseAbstract();
```

Comment:
I think we have to export this class, it's providing inheritance from EventEmitter